sources/packages/api/src/services/archive-db/archive-db.service.ts

```diff
@@ -0,0 +1,39 @@
+import { Injectable } from "@nestjs/common";
+import { createConnection, Connection } from "typeorm";
+import { PostgresConnectionOptions } from "typeorm/driver/postgres/PostgresConnectionOptions";
+
+const config: PostgresConnectionOptions = require("../../../ormconfig");
+
+@Injectable()
+export class ArchiveDbService {
+  private _connection?: Connection;
+
+  private async init() {
+    try {
+      this._connection = await createConnection({
```

Comment:
You are not setting the db name here, right?
Is it going to be dedicated to sfaz DB or generic for any other raw SQL query to any other possible db?

Reply:
This was wrong, I will fix this.